clang/test/CIR/CodeGen/fullexpr.cpp

```diff
@@ -1,5 +1,8 @@
 // RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
 // RUN: FileCheck --input-file=%t.cir %s
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm -o - %s \
```

Comment:
I thing we should check the `-emit-cir-flat` output instead, since it more directly tests the code you're changing.

Reply:
that's good idea. I'll add that as part of test.